tests/001-test-redis.sh

```diff
@@ -49,6 +49,19 @@ test_redis_nomad_job() {
        exit 1
     fi
 
+    echo "INFO: Check if memory and memory_max are set correctly in the cgroup filesystem."
+    task_name=$(sudo CONTAINERD_NAMESPACE=nomad ctr containers ls|awk 'NR!=1'|cut -d' ' -f1)
+    memory_soft_limit=$(sudo cat /sys/fs/cgroup/memory/nomad/$task_name/memory.soft_limit_in_bytes)
+    if [ $memory_soft_limit != "268435456" ]; then
+       echo "ERROR: memory should be 256 MB."
```

Comment:
May I suggest using bash arithmetic here, hopefully to make understanding the test a bit easier:

```suggestion
    if [ $memory_soft_limit != "$(( 256 * 1024 * 1024 ))" ]; then
       echo "ERROR: memory should be 256 MB. Found ${memory_soft_limit}"
```

Reply:
Fixed.